site/admin/src/home/home-use-case.js

```diff
@@ -28,6 +28,11 @@ export const deleteDevice = (dispatch) => async (device) => {
     await fetchDevices(dispatch)()
 }
 
+export const deleteTopic = (dispatch) => async (topic) => {
+    await removeTopic(topic.id)
+    await fetchTopics(dispatch)()
```

Comment:
we don't currently watch any database paths, so we're manually refreshing the topics after deleting